client/src/api/schema/schema.ts

```diff
@@ -9020,6 +9026,163 @@ export interface components {
              */
             short_term_storage_request_id: string;
         };
+        /**
+         * ShowJobResponse
+         * @description Basic information about a job.
+         */
+        ShowJobResponse: {
+            /**
+             * Command Line
+             * @description The command line produced by the job. Users can see this value if allowed in the configuration, administrator can always see this value.
+             */
+            command_line?: string;
+            /**
+             * Command Version
+             * @description Tool version indicated during job execution.
+             */
+            command_version: string;
+            /**
+             * Copied from Job-ID
+             * @description Reference to cached job if job execution was cached.
+             * @example 0123456789ABCDEF
+             */
+            copied_from_job_id?: string;
+            /**
+             * Create Time
+             * Format: date-time
+             * @description The time and date this item was created.
+             */
+            create_time: string;
+            /**
+             * Job dependencies
+             * @description The dependencies of the job.
+             */
+            dependencies?: Record<string, never>[];
+            /**
+             * Exit Code
+             * @description The exit code returned by the tool. Can be unset if the job is not completed yet.
+             */
+            exit_code?: number;
+            /**
+             * External ID
+             * @description The job id used by the external job runner (Condor, Pulsar, etc.)Only administrator can see this value.
+             */
+            external_id?: string;
+            /**
+             * Galaxy Version
+             * @description The (major) version of Galaxy used to create this job.
+             * @example 21.05
+             */
+            galaxy_version: string;
```

Comment:
This needs to be optional, that hasn't always been present.

Reply:
Should I keep `galaxy_version` in  `ATTRIBUTES` - like suggested by @davelopez above - now that the field is optional?

Reply:
yes

Reply:
it currently doesn't seem to be optional though ?